<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pesquisar</class>
 <widget class="QDialog" name="Pesquisar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="Fundo">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>20</y>
     <width>791</width>
     <height>501</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>381</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Horario de Ida</string>
   </property>
   <widget class="QLabel" name="Fundo2">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>10</y>
      <width>371</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Partida   Cidade    Estacao    Chegada     Cidade      Estacao    Preco</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>90</y>
     <width>361</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Horario de Volta</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Comprar">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Selecionar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Sair">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
  </widget>
  <zorder>checkBox_4</zorder>
  <zorder>Fundo</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>Comprar</zorder>
  <zorder>Sair</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
